src/openforms/registrations/contrib/objects_api/tests/test_backend.py

```diff
@@ -0,0 +1,221 @@
+        self.assertEqual(create_object.method, "POST")
+        self.assertEqual(create_object.url, "https://objecten.nl/api/v2/api/v2/objects")
+        self.assertDictEqual(create_object_body, expected_body)
+
+    def test_submission_with_objects_api_backend_use_config_defaults(self, m):
+        submission = SubmissionFactory.from_components(
+            [
+                {
+                    "key": "voornaam",
+                    "registration": {
+                        "attribute": RegistrationAttribute.initiator_voornamen,
+                    },
+                },
+                {
+                    "key": "achternaam",
+                    "registration": {
+                        "attribute": RegistrationAttribute.initiator_geslachtsnaam,
+                    },
+                },
+                {
+                    "key": "tussenvoegsel",
+                    "registration": {
+                        "attribute": RegistrationAttribute.initiator_tussenvoegsel,
+                    },
+                },
+                {
+                    "key": "geboortedatum",
+                    "registration": {
+                        "attribute": RegistrationAttribute.initiator_geboortedatum,
+                    },
+                },
+            ],
+            submitted_data={
+                "voornaam": "Foo",
+                "achternaam": "Bar",
+                "tussenvoegsel": "de",
+                "geboortedatum": "2000-12-31",
+            },
+            bsn="111222333",
+        )
+
+        mock_service_oas_get(m, "https://objecten.nl/api/v2/", "objecten")
+
+        expected_result = {
+            "url": "https://objecten.nl/api/v1/objects/1",
+            "uuid": "095be615-a8ad-4c33-8e9c-c7612fbf6c9f",
+            "type": "https://objecttypen.nl/api/v1/objecttypes/1",
+            "record": {
+                "index": 0,
+                "typeVersion": 1,
+                "data": {
+                    "data": submission.get_merged_data(),
+                    "type": "terugbelnotitie",
+                    "submission_id": str(submission.uuid),
+                },
+                "geometry": {"type": "Point", "coordinates": [0, 0]},
+                "startAt": date.today().isoformat(),
+                "endAt": date.today().isoformat(),
+                "registrationAt": date.today().isoformat(),
+                "correctionFor": 0,
+                "correctedBy": "",
+            },
+        }
+
+        m.post(
+            # FIXME zgw-consumers seems to duplicate /api/v2
+            "https://objecten.nl/api/v2/api/v2/objects",
+            status_code=201,
+            json=expected_result,
+        )
+
+        plugin = ObjectsAPIRegistration("objects_api")
+        result = plugin.register_submission(submission, {})
+
+        # Result is simply the created object
+        self.assertEqual(result, expected_result)
+
+        # self.assertEqual(len(m.request_history), 1)
```

Comment:
would be the calls to fetch the API spec and the calls to do the creation, I'd like an explicit assertion here